helm/hpcc/templates/roxie.yaml

```diff
@@ -113,9 +113,9 @@ spec:
   template:
     metadata:
       labels:
+{{ include "hpcc.addStandardLabels" (dict "root" $ "component" "roxie" "name" $roxie.name "instance" $commonCtx.toponame) | indent 8 }}
```

Comment:
I was expecting this to be component=topology-server, name=roxie, instance=<toponame> ?